src/aks-preview/azext_aks_preview/_consts.py

```diff
@@ -25,6 +25,10 @@
 CONST_INGRESS_APPGW_SUBNET_PREFIX = "subnetPrefix"
 CONST_INGRESS_APPGW_WATCH_NAMESPACE = "watchNamespace"
 
+# Open Service Mesh configuration keys
+CONST_OSM_ADDON_NAME = "openServiceMesh"
```

Comment:
I would prefer we, like the commend above, use a full name here. So instead of OSM call it OPEN_SERVICE_MESH. This is doubly true since we have another addon called OMS already and I think it would be easy to mistake the two if you are reading quickly

Reply:
Done